rules/sql-server-cpu-pressure/rule.md

```diff
@@ -0,0 +1,126 @@
+---
+type: rule
+archivedreason: 
+title: What to do about SQL Server CPU Pressure?
+guid: 870f4673-3a45-48d9-b73d-14921a77bc70
+uri: sql-performance-cpu-pressure
+created: 2023-12-27T07:26:54.0000000Z
+authors:
+  - title: Bryden Oliver
+    url: https://ssw.com.au/people/bryden-oliver
+related: 
+  - identify-sql-performance-sql-server
+  - identify-sql-performance-azure
+  - sql-server-memory-pressure
+  - sql-server-io-pressure
+---
+So you've identified that your SQL Server is under CPU pressure. What can you do about it?
+
+<!--endintro-->
+
+Here's a simple in-depth presentation covering things that can help reduce CPU pressure.
+
+1. Should you throw hardware at the problem
```

Comment:
**issue**: Grammar - the first point is written as a question (without question mark), while the remaining points are actions.
```suggestion
1. Throw hardware at the problem
```